dags/veda_data_pipeline/utils/s3_discovery.py

```diff
@@ -111,12 +111,10 @@ def construct_single_asset_items(discovered_files: List[str]) -> dict:
     for uri in discovered_files:
         # Each file gets its matched asset type and id
         filename = uri.split("/")[-1]
-        filename_without_extensions = Path(filename)
-        while filename_without_extensions.suffix: #for filenames that have multiple extensions i.e. *.tif.cog
-             filename_without_extensions=filename_without_extensions.with_suffix('')
+        filename_without_extension = Path(filename).stem
```

Comment:
Do we care about the collection `modis-fire-anomalies-diff-covid-19-changing-landscape`? The item feature.id string `MODIS-AF_anomalies_USA_202010_M10_0.25-dashboard` would be `MODIS-AF_anomalies_USA_202010_M10_0`. Is that okay, since we might not need to support that collection? 🤔 

https://staging-stac.delta-backend.com/collections/modis-fire-anomalies-diff-covid-19-changing-landscape/items

Reply:
I had no idea we had so many of weird filenames! Are we not supporting it or it's not used in the dashboard currently?

Reply:
from a cursory search it doesn't look like it's used in the dashboard, but I could be wrong! @anayeaye , do you know?

Reply:
https://github.com/NASA-IMPACT/veda-config/tree/develop/datasets I'm guessing since it's not here, we're good